src/translations/spreadsheet-en.ts

```diff
@@ -32,6 +32,10 @@ const spreadsheetEn = {
     'spreadsheet/custom_column/column_name_already_exist': 'Column name already exists',
     'spreadsheet/custom_column/dependencies': 'The formula is linked to other columns',
     'spreadsheet/custom_column/nodes': 'Nodes',
+    'spreadsheet/custom_column/option/parameter': 'Parameter',
```

Comment:
nit: configure ?

Reply:
I wanted to be consistent with the dialog title:

'spreadsheet/custom_column/parameter_nodes': 'Parameter nodes aliases',

Reply:
Yes but maybe we should change them all, parameter is not a verb in english ^^